std_msgs/Header          header                # header

int32                    edge_id               # the id of the edge

int32                    source                # the id of the source node

int32                    target                # the id of the target node

geometry_msgs/PointStamped[]  line             # points along the edge

float64                  length                # length of edge  
